scripts/servermanager.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+# Stop on errors, comment in, if needed
+#set -e
+
+### Includes
+source /includes/config.sh
+source /includes/cron.sh
+source /includes/rcon.sh
+source /includes/security.sh
+source /includes/webhook.sh
+source /includes/colors.sh
+
+steamcmd_dir="/home/steam/steamcmd"
+
+### Server Functions
+
+function start_server() {
+    # IF Bash extension used:
+    # https://stackoverflow.com/a/13864829
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02
+
+    pp --success "\n>>> Starting the gameserver\n"
+    cd "$GAME_ROOT" || exit
```

Comment:
You created that in dockerfile and entrypoint, if that would error the pipeline would have already failed. Why check it again?